azure/basic-arm-template/xebialabs/xld-apps.yaml.tmpl

```diff
@@ -0,0 +1,25 @@
+{{$app := .AppName | kebabcase}}
+apiVersion: xl-deploy/v1
+kind: Applications
+spec:
+- name: {{$app}}
+  type: core.Directory
+  children:
+  - name: {{$app}}-application
+    type: udm.Application
+    children:
+    - name: '1.0.0'
+      type: udm.DeploymentPackage
+      deployables:
+      - name: {{$app}}-template
+        type: azure.arm.Template
+        deploymentMode: Incremental
+        resourceGroupName: {{.ResourceGroup}}
+        location: {{.ResourceGroupLocation}}
+        deploymentName: {{.AppName}}-deployment
+        file: !file "../arm/template.json"
+        inputVariables:
+          adminUsername: {{.AdminUsername}}
+          adminPassword: !value AdminPassword
```

Comment:
This password is been requested as a SecretInput but then is being put here as plain text. Is not better to use an encrypted dictionary in this case??

Reply:
The password comes from the `secrets.xlvals` file. I think this is an accepted standard. I am working on a blueprint that showcases encrypted dictionaries though.